### ISSUE TITLE
SlashBlade incompatibility (NoClassDefFoundError org/lwjglx/util/mapped/CacheUtil)

### ISSUE THREAD

**Issue by LemonNick** [GTNewHorizons/lwjgl3ify#92]
Crash-report:

```
cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: org/lwjglx/util/mapped/CacheUtil
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:559)
	at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:165)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:29)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at pro.gravit.utils.launch.ModuleLaunch.launch(Unknown Source)
	at pro.gravit.launcher.client.ClientLauncherEntryPoint.launch(Unknown Source)
	at pro.gravit.launcher.client.ClientLauncherEntryPoint.realMain(Unknown Source)
	at pro.gravit.launcher.client.ClientLauncherEntryPoint.main(Unknown Source)
Caused by: java.lang.NoClassDefFoundError: org/lwjglx/util/mapped/CacheUtil
	at mods.flammpfeil.slashblade.ItemRendererBaseWeapon.<init>(ItemRendererBaseWeapon.java:171)
	at mods.flammpfeil.slashblade.InitProxyClient.initializeItemRenderer(InitProxyClient.java:52)
	at mods.flammpfeil.slashblade.SlashBlade.preInit(SlashBlade.java:240)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
	... 13 more
Caused by: java.lang.ClassNotFoundException: org.lwjglx.util.mapped.CacheUtil
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:134)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:593)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 37 more
```

**Comment by mts2200** [comment]
Hello! I join issue. Here is the full log from the multimc client: https://pastebin.com/7XEQznMx
Can u fix it in lwjgl3ify or Hodpepodge?

**Comment by eigenraven** [comment]
Should be fixed in 2.0.0-alpha-6
